### ISSUE TITLE
add sortbytag option in track configuration

### ISSUE THREAD

**Issue by roelnoten** [igvteam/igv.js#1001]
We need the ability to have bam alignments sorted by a tag upon opening of an igv session. This functionality is now available (as per https://github.com/igvteam/igv.js/issues/969) in the track popup menu but that's to dificult for our users and they typically don't know on which tag to sort.

**Comment by jrobinso** [comment]
I pushed some fixes to allow sort-by-tag initialization.  This will be in the next point release (2.3.4).  Example follows

```
   const options =
        {
            genome: "hg19",
            locus: "chr8:128,750,968-128,751,006",
            tracks: [
                {
                    type: 'alignment',
                    format: 'cram',
                    url: 'https://s3.amazonaws.com/1000genomes/phase3/data/HG00096/exome_alignment/HG00096.mapped.ILLUMINA.bwa.GBR.exome.20120522.bam.cram',
                    indexURL: 'https://s3.amazonaws.com/1000genomes/phase3/data/HG00096/exome_alignment/HG00096.mapped.ILLUMINA.bwa.GBR.exome.20120522.bam.cram.crai',
                    name: 'HG00096',
                    colorBy: "tag",
                    colorByTag: "XC",
                    sort: {
                        locus: "chr8:128,750,986",
                        direction: "ASC",
                        option: "TAG",
                        tag: "XC"
                    },
                }

            ]
        };
```

**Comment by jrobinso** [comment]
This is released, version 2.3.5.

**Comment by roelnoten** [comment]
Thanks! It works. Just in time for inclusion for our release ;)
